launch/autoscale-by-metric.html.markerb

```diff
@@ -44,10 +44,10 @@ fly secrets set -a my-autoscaler --stage FAS_API_TOKEN="FlyV1 ..."
 The next auth token you'll need is one that has permissions to read from your organization's Prometheus data on Fly:
 
 ```cmd
-fly tokens create readonly -o my-org
```

Comment:
Does specifying an org without -o work? I suspect we were trying to avoid forcing people into an interactive CLI in case they wanted to use this for scripts that won't have interactive shell.

Reply:
Ahh, it does - let me update this to add that, thanks!